packages/botonic-api/src/bot-dispatcher/aws-bot-dispatcher.ts

```diff
@@ -0,0 +1,39 @@
+import { ApiGatewayManagementApi } from 'aws-sdk'
+
+import { BotDispatcher, DispatchArgs } from '.'
+
+export class AWSBotDispatcher implements BotDispatcher {
+  bot
+  constructor(bot) {
+    this.bot = bot
+  }
```

Comment:
Question: what principles are you following in this implementation?
- I can see a `ts` file, but only a few types declared (what type is this bot?)
- I doesn't look that the formatting of this file is working. There are blank lines between declarations missing

Reply:
* The type that should go with bot is declared in `@botonic/react`, which means adding an extra dependency just for one type. In next iterations, we should see where these types will reside (ideally in @botonic/core)
* I will review all the configurations for the new packages we have created so far at the same time. Just created a task for that: https://linear.app/hubtype/issue/BOT-598/homogenize-prettier-eslint-config-etc-for-new-botonic-packages